string stm32_movement
int64 stm32_heading
int64 stm32_depth